drivers/wifi/siwx917/Kconfig.siwx917

```diff
@@ -21,6 +21,11 @@ config WIFI_SIWX917
 
 if WIFI_SIWX917
 
+# WiseConnect create threads with realtime priority. Default (10kHz) clock tick
+# prevent proper use of the system with these threads.
+config SYS_CLOCK_TICKS_PER_SEC
+	default 1024
```

Comment:
I think 1000 would make more sense (most new WiseConnect examples are also using 1000).

Reply:
That seems strange, considering a 32768 Hz clock divided by 32 would give 1024. Are they not running on a low frequency timer (we are not _yet_, but were planning to)?